internal/pkg/service/common/httpserver/middleware/otel_endpoint.go

```diff
@@ -24,7 +25,7 @@ func OpenTelemetryExtractEndpoint() GoaMiddleware {
 			if endpointName != "" {
 				// Set metrics attributes
 				labeler, _ := otelhttp.LabelerFromContext(ctx)
-				labeler.Add(attribute.String("endpoint.name", endpointName))
+				labeler.Add(attribute.String("endpoint.name", strhelper.NormalizeName(endpointName))) // normalize name: prometheus converts string to lower
 
```

Comment:
Prometheus/DD prevadza nazvy attrs na lowercase, ... takze namiesto `MyEndpoint` som mal `myendpoint`, ... pri dlhsich nazvoch sa to blbo citalo, ... tak pouzijem funkciu, ktoru uz mame (aj s testami) na prevod na `my-endpoint`.